plugins/ssr/ssripats.ml

```diff
@@ -137,17 +235,68 @@ let tacCHECK_HYPS_EXIST hyps = Goal.enter begin fun gl ->
   tclUNIT ()
 end
 
+let name_seeds_of_ind env sigma ty =
```

Comment:
Can you explain what this does in a comment?

Reply:
done